eZ/Bundle/EzPublishLegacySearchEngineBundle/Command/LseCreateIndexCommand.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+/**
+ * This file is part of the eZ Publish Kernel package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishLegacySearchEngineBundle\Command;
+
+use eZ\Publish\Core\Persistence\Database\DatabaseHandler;
+use eZ\Publish\SPI\Persistence\Content;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Helper\ProgressHelper;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use eZ\Publish\SPI\Persistence\Content\ContentInfo;
+use eZ\Publish\Core\Base\Exceptions\NotFoundException;
+use eZ\Publish\Core\Search\Legacy\Content\Handler as SearchHandler;
+use RuntimeException;
+use PDO;
+
+/**
+ * Console command ezplatform:lse_create_index indexes content objects for legacy search engine.
+ */
+class LseCreateIndexCommand extends ContainerAwareCommand
```

Comment:
@andrerom: Ok. What should be the name of this command then? I think `ezplatform:create_index` is too ambiguous. `ezplatform:legacy_search_engine_create_index` seems too long. 
Could it be `ezplatform:create_sql_search_index`?